### ISSUE TITLE
The audio recordings have an invalid duration field in the wav file.

### ISSUE THREAD

**Issue by SamuelLarkin** [EveryVoiceTTS/common-voice#5]
Internally, Common-Voice uses [stream-transcoder.js](https://www.npmjs.com/package/stream-transcoder) to process the audio recording sent by the browser of the user.  Basically, it unwraps the user-sent audio file from its ogg or whatever the container the user's browser sends the audio in into a raw wav file (16-bit Signed Integer PCM).  Know that `stream-transcoder` is a wrapper on top of `ffmpeg`.  The duration header gets replaced by a generic duration by `ffmpeg` because Common-Voice calls `stream-transcoder` in a streaming mode.  Here's an example of a call `ffmpeg -i - -acodec pcm_s16le -f wav -ac 1 -y pipe:1` where `-i -` means the input is piped in and `pipe:1` means the output is also piped to the next program without being written to disk.  Once the output has been streamed/piped out, that is when the duration of the input stream is known and then, it is too late to go back and write the duration header.

**Comment by SamuelLarkin** [comment]
# Proposed solution
Change how Common-Voice invoke `steam-transcoder` to write the output file into a temporary directory.  Then send that temporary file to S3.

[stream-transcoder API](https://www.npmjs.com/package/stream-transcoder#transcoderwritetofilefile)
```
transcoder.writeToFile(file)
file String - Path of filename.
Writes transcoded media data to file.

Returns transcoder object
```

The new call to process the audio will translate to:
```sh
ffmpeg \
  -i - \
  -acodec pcm_s16le \
  -f wav \
  -ac 1 \
  -y /tmp/8f7b3457-1a9f-4c9e-a44c-d33fadb3bb5c_00dc3e8d42d07163e93c06054b7ae4b58c0672820a89d07dc82cc4be0f38c7a88gDWp6/output.wav
```

**Comment by SamuelLarkin** [comment]
Solved by #3.
